packages/replay/README.md

```diff
@@ -96,24 +90,37 @@ Alternatively, rather than recording an entire session, you can capture a replay
 
 Sampling allows you to control how much of your website's traffic will result in a Session Replay. There are two sample rates you can adjust to get the replays more relevant to your interests:
 
-- `sessionSampleRate` - The sample rate for replays that begin recording immediately and last the entirety of the user's session.
-- `errorSampleRate` - The sample rate for replays that are recorded when an error happens. This type of replay will record up to a minute of events prior to the error and continue recording until the session ends.
+- `replaysSampleRate` - The sample rate for replays that begin recording immediately and last the entirety of the user's session.
+- `replaysOnErrorSampleRate` - The sample rate for replays that are recorded when an error happens. This type of replay will record up to a minute of events prior to the error and continue recording until the session ends.
 
-Sampling occurs when the session is first started. `sessionSampleRate` is evaluated first. If it is sampled, then the replay recording begins. Otherwise, `errorSampleRate` is evaluated and if it is sampled, the integration will begin buffering the replay and will only upload a replay to Sentry when an error occurs. The remainder of the replay will behave similarly to a whole-session replay.
+Sampling occurs when the session is first started. `replaysSampleRate` is evaluated first. If it is sampled, then the replay recording begins. Otherwise, `replaysOnErrorSampleRate` is evaluated and if it is sampled, the integration will begin buffering the replay and will only upload a replay to Sentry when an error occurs. The remainder of the replay will behave similarly to a whole-session replay.
 
 
 ## Configuration
 
-### General Configuration
+### SDK Configuration
+
+The following options can be configured on the root level of your browser-based Sentry SDK, in `init({})`:
+
+
+| key                 | type    | default | description                                                                                                                                                                                                                     |
```

Comment:
The description column feels redundant because we describe both sample rates immediately above. I wonder if we should keep what is noted in the parentheses and remove the rest? Or we at least repeat the same definitions as above + what's in the parentheses?

Reply:
I think it makes sense to still keep this, but maybe rewrite it a bit - I'll give it a try!

Reply:
🙏🏻

Reply:
Let me know if you think this is better - I copied the main text from above, and added the "0-1" info as a regular sentence.

Reply:
Yes! Previously, the description referred to the sample rates as sampling 'sessions', and the other section said the sample rates were sampling 'replays', so now this reads clearer :)